llmstudio/engine/providers/azure.py

```diff
@@ -65,5 +65,5 @@ async def parse_response(
                     and chunk.choices[0].delta.content is not None
                 ):
                     yield chunk.choices[0].delta.content
-            except Exception as e:
+            except Exception:
```

Comment:
dont we need to log this error? it's super broad and can lead to silent errors

Reply:
i just removed it, as there was no need